test/e2e/framework/util.go

```diff
@@ -260,13 +261,29 @@ func log(level string, format string, args ...interface{}) {
 	fmt.Fprintf(GinkgoWriter, nowStamp()+": "+level+": "+format+"\n", args...)
 }
 
+// Used for important logs, controlled by ginkgo, on by default
 func Logf(format string, args ...interface{}) {
 	log("INFO", format, args...)
 }
 
+// Less important logs, controlled with -v
+var totaldebugs = 0
+
+// Debugf suppresses extra logging, emits certain debug logs with a usage message if there is a lot of information being
+// suppressed.
+func Debugf(format string, args ...interface{}) {
```

Comment:
You know something, the whole code-base could use a log leaning + abstraction layer.  This was one of @spiffxp issues he ran into a while ago, the system back'd up on what "should have been" clearly defined Debug logs.